frontend/src/components/payment/page/PaymentPage.tsx

```diff
@@ -65,8 +62,8 @@ const PaymentPage: FC = () => {
   const handleCallBack = (stats: boolean) => {
     if (!stats) {
       toggle();
-      if (isPayment) history.replace({ pathname: '/' });
-      if (isCancel) history.replace({ pathname: '/' });
+      if (isPayment) history.push({ pathname: '/' });
+      if (isCancel) history.push({ pathname: '/' });
```

Comment:
Can combine this to `if (isPayment || isCancel) ...`

Reply:
this condition for future change to orderdetail page after payment